addons/mail/static/src/core/common/thread_service.js

```diff
@@ -432,6 +432,9 @@ export class ThreadService {
      * @param {Object} [options]
      */
     open(thread, replaceNewMessageChatWindow, options) {
+        if (thread.type === "chat") {
+            this.env.services.im_status.fetch();
+        }
```

Comment:
I'm not sure if its the "correct" open method.
Don't we want the lowest level, `setDiscussThread` for discuss app, and something else for chat window?

Reply:
In master we tried to enforce the usage of thread.open I think? I'll check for 17.0 but that's sad to call it on each open method while it should conceptually be a single call to thread.open.